.github/workflows/ci.yml

```diff
@@ -0,0 +1,107 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🛑 Cancel Previous Runs
```

Comment:
Can we remove this?

Even if there is a previous build, I would still expect them to be released separately.